### DISCUSSION TITLE
Privategpt with c# .net?

### DISCUSSION THREAD

**Discussion by ranasikandar** [imartinez/privateGPT#702] [Q&A] [answered]
Is there anything that .net developers can integrate privategpt in .net projects? 
I have been googling for days but couldn't find anything.

**Comment by ArronHoltzhausen** [comment, marked as answer]
well their api looks comprehensive 

https://docs.privategpt.dev/api-reference/overview/api-reference-overview

you should be able to use that easily in your .net project
